### PR TITLE
ci: auto-commit ci changes

### LINKED ISSUE #5: ci: pre-commit ci imitation - run pre-commit, commit changes

### PR DESCRIPTION
**Pull request by MartinBernstorff**
Fixes #5.

Probably want to align the token name with #6 so we only need to setup one PAT for repo setup.

### CODE REVIEW

**Comment by MartinBernstorff** [comment]
Test repo here: https://github.com/MartinBernstorff/commit-precommit-changes/pull/7

File: {{cookiecutter.project_name}}/.pre-commit-config.yaml
Comment (KennethEnevoldsen) [inline {{cookiecutter.project_name}}/.pre-commit-config.yaml:1]: Why manual? For me that seems equivalent to commit without an install?

Comment (MartinBernstorff) [reply {{cookiecutter.project_name}}/.pre-commit-config.yaml:1]: You mean running the hooks without an install? Sure! Since the changes from the hooks are now applied by the action, my preference would be for the hooks to only run on manual invocation - but open to talking about it!

Comment (MartinBernstorff) [reply {{cookiecutter.project_name}}/.pre-commit-config.yaml:1]: I think we need to examine the PATs as well - I worry they might be getting permissions to write to the repo. 

If that's the case, someone could submit a PR, echo the PAT, then use that to modify files on the core repo. That would be a huge security vulnerability.

Comment (MartinBernstorff) [reply {{cookiecutter.project_name}}/.pre-commit-config.yaml:1]: We are somewhat protected by actions only being run after approval by a maintainer. 

It does require that we examine whether any workflows have been modified before running the workflows, another manual step, but it eliminates the risk of automatic vulnerability induction.

**Review by KennethEnevoldsen — APPROVED** [review]

**Pull request merged**
